pkgs/by-name/xe/xeus/package.nix

```diff
@@ -11,13 +11,13 @@
 
 stdenv.mkDerivation rec {
```

Comment:
Please use [`finalAttrs`](https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes) instead, that makes overriding easier.

Reply:
Will do!

For the `src = fetchFromGitHub` block, is the preferred syntax:
1. ```
    repo = pname;
    tag = version;
    ```
or

2. ```
     repo = finalAttrs.pname;
     tag = finalAttrs.version;
    ```

?

(I'm pushing up with version 1 for now.)

Reply:
_Edit_: Changed to version 2.